tests/texture/stargz/estargz_sample.json

```diff
@@ -0,0 +1,50 @@
+{
```

Comment:
Is this file specially added?

Reply:
The builder has a ut case, which uses this file, though it's ignored currently.
```
   #[ignore]
    #[test]
    fn test_build_stargz_toc() {
        let tmp_dir = vmm_sys_util::tempdir::TempDir::new().unwrap();
        let tmp_dir = tmp_dir.as_path().to_path_buf();
        let root_dir = &std::env::var("CARGO_MANIFEST_DIR").expect("$CARGO_MANIFEST_DIR");
        let source_path =
            PathBuf::from(root_dir).join("../tests/texture/stargz/estargz_sample.json");
```